resources/geocoding/en/98.txt

```diff
@@ -13,9 +13,7 @@
 # limitations under the License.
 
 # Generated from:
-# http://www.itu.int/oth/T0202000066/en [2012-07-24]
```

Comment:
I would recommend leaving all the old URLs for posterity's sake - especially the original ITU link, which is a stable URL where updates can usually (hopefully) be tracked consistently.